src/App.js

```diff
@@ -1,19 +1,37 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import DefaultUI  from './components/materialUI';
+import Register from './components/register';
+import Login from './components/login';
+import Paper from 'material-ui/Paper';
+//import './mystyles.css';
+
 import './App.css';
 
+const style = {
```

Comment:
Always put your styles in a CSS folder

Reply:
This shall be improved in upcoming PRs.Thanks for the feedback